sdk/communication/azure-communication-chat/src/main/java/com/azure/android/communication/chat/ChatAsyncClient.java

```diff
@@ -206,6 +214,17 @@ public ChatAsyncClient build() {
             if (credentialInterceptor != null) {
                 serviceClientBuilder.setCredentialsInterceptor(credentialInterceptor);
             }
+            if (userAgentInterceptor == null) {
+                userAgentInterceptor = new UserAgentInterceptor(
+                    null,
+                    "azure-communication-chat",
+                    "1.0.0-beta.3",
+                    null,
+                    null,
+                    null);
+            }
```

Comment:
I believe there is standard way of formatting the agent string, c.f. https://azure.github.io/azure-sdk/general_azurecore.html and search for "User-Agent header"

Reply:
You should use the 3 or 4-parameter versions of the constructor for this, and make sure to pass a valid `Context`.